WooCommerce/WooCommerceTests/Mockups/MockPushNotificationsManager.swift

```diff
@@ -2,6 +2,7 @@
 import Foundation
 import UIKit
 @testable import WooCommerce
+@testable import Yosemite
```

Comment:
I don't think we need the `@testable`

```suggestion
import Yosemite
```

Reply:
updated in https://github.com/woocommerce/woocommerce-ios/pull/2320/commits/bb9b20151fad7435498c43f544ba7ba7db59bb78